tests/test_agent.py

```diff
@@ -40,7 +40,6 @@ def _handler_that_errors(event, context):
     return iopipe, _handler_that_errors
 
 
-# N.B. this must be the first test!
 def test_coldstarts(handler, mock_context, monkeypatch):
```

Comment:
This doesn't have to be the first test anymore? (I see a comment deletion)

Reply:
My comment has disappeared, so adding again. The `monkeypatch` handles the setup for this test.

Reply:
@kolanos should this comment exist or not? It's still deleted (line 43 in the old file)

Reply:
The test is now deterministic and can be run in any order with other tests. So that's why the comment was removed.